share/styles/chords_jazz.xml

```diff
@@ -146,7 +146,7 @@
 
     </font>
 
-  <font class="modifer" family="MuseJazz Text">
+  <font class="modifier" family="MuseJazz Text">
```

Comment:
I checked and saw that there wasn't an actual class named `modifer` but I did find a whole lot of classes named `modifiers`